app/lib/efile/md/md502_calculator.rb

```diff
@@ -289,7 +281,12 @@ def calculate_line_16
       }
 
       def calculate_deduction_method
-        gross_income_amount = @intake.tax_calculator.gross_income_amount
+        gross_income_amount = if @direct_file_data.claimed_as_dependent?
```

Comment:
so was that calling a different method `gross_income_amount` on a different object?

Reply:
when you try to to call the method on the intake, the lines havent been transformed to the intake that are needed for it. the issue is the circular dependency where gross_income_amount is being called before the lines are set.